src/main/java/mate/academy/springbootstore/service/user/UserServiceImpl.java

```diff
@@ -38,15 +36,15 @@ public UserResponseDto register(UserRegistrationRequestDto request)
         User user = userMapper.toEntity(request);
         user.setPassword(passwordEncoder.encode(request.getPassword()));
 
-        Role.RoleName defaultRole = Role.RoleName.USER;
-        Role userRole = roleRepository.findByName(defaultRole)
-                .orElseThrow(() ->
-                        new RuntimeException("Role " + defaultRole + " not found")
-                );
+        Role userRole = roleRepository.findByName(Role.RoleName.USER)
+                .orElseThrow(() -> new RuntimeException("Role USER not found"));

```

Comment:
use Role.RoleName.USER in message
```suggestion
                .orElseThrow(() -> new EntityNotFoundException("Role USER not found"));
```